src/components/UpcomingEventsList.tsx

```diff
@@ -1,39 +1,30 @@
 import { useEffect, useState } from "react"
+import { useTranslation } from "next-i18next"
 import { Box } from "@chakra-ui/react"
 
-import Translation from "@/components/Translation"
+import type { CommunityConference } from "@/lib/types"
+
+import { Button } from "@/components/Buttons"
+import EventCard from "@/components/EventCard"
+import InfoBanner from "@/components/InfoBanner"
+import InlineLink from "@/components/Link"
 
 import { trackCustomEvent } from "@/lib/utils/matomo"
 
-import events from "../data/community-events.json"
-
-import { Button } from "./Buttons"
-import EventCard from "./EventCard"
-import InfoBanner from "./InfoBanner"
-import InlineLink from "./Link"
-
-interface ICommunityEventData {
-  title: string
-  to: string
-  sponsor: string | null
-  location: string
-  description: string
-  startDate: string
-  endDate: string
-}
+import communityConferences from "@/data/community-events"
 
-interface IOrderedUpcomingEventType extends ICommunityEventData {
+type OrderedUpcomingEvent = CommunityConference & {
   date: string
   formattedDetails: string
 }
 
-const UpcomingEventsList: React.FC = () => {
+const UpcomingEventsList = () => {
+  const { t } = useTranslation("page-community")
```

Comment:
<img width="919" alt="Screenshot 2024-01-24 at 17 15 42" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/6826151b-51f4-4209-a620-fe41336f8ece">

Found a string for an empty set that was missing its namespace prefix... went ahead and swapped this to use `useTranslation` instead of the `Translation` component.